addressing issue #26 (mapping RPP and EPP as deliverable)

anewton1998 opened the pull request:
This PR is add mapping documents to the deliverables section. See Issue #26

mwullink commented on rpp-charter.md:
```diff
@@ -64,6 +64,7 @@ After finding consensus regarding requirements, the next documents to be deliver
 by this working group will describe the core architecture and JSON mapping of RPP with regard to the
 provisioning of domain names, followed by any specifications necessary
 to implement the core architecture and the JSON mapping for the different objects to be provisioned.
+These documents may then be followed by mappings between RPP and EPP.
```
I think the "mappings between RPP and EPP" document must be moved.
new proposal:

"this working group will describe the core architecture and JSON mapping of RPP with regard to the
provisioning of domain names. These documents are then followed by a document describing the mappings between RPP and EPP. Followed by any specifications necessary to implement the core architecture and the JSON mapping for the different objects to be provisioned."

Rationale:
It may take some time (if ever) before other objects (numbers? widgets?) are described in documents, implementers need a mapping between RPP and EPP after the core architecture and JSON mapping of RPP  for domains names is completed.

anewton1998 replied:
How does a mapping occur before any core specifications? It seems to me that if you want to map "blah": "foo" in JSON to <blah>foo</blah> in XML, you need concrete implementation specifications.

mwullink replied:
no, i don't mean to say that the mapping from RPP to EPP comes first.

i think this is the correct order:

1: the core architecture and JSON mapping of RPP with regard to the provisioning of domain names
2: mappings between RPP and EPP.
3: any other (optional) object mapping, e.g. numbers.


because at the moment it is not clear that other object mapping (3) will ever happen and if it does how long this will take. Therefore, it's better to create the  mappings between RPP and EPP (2) immediately after the core architecture and JSON mapping of RPP (1)

mwullink reviewed (commented):
see inline comment

pawel-kow commented:
I think putting things in some order of dependencies is kind of void discussion when it comes to a charter.
The core question seems to be whether the RPP working group shall define a mapping between RPP and EPP as a deliverable.
If we say yes, then it's of secondary importance when exactly the mapping document would be ready.
The working group will likely keep it in mind already doing the architecture and defining the protocol on every level and it's enough to have a consensus that mapping is possible without yet producing a document defining it.

I would be in favour of adding such deliverable without determining the order of those now.

mwullink commented:
The wg should deliver an RPP -> EPP mapping document but i  agree with Pawel that  removing the ordering from the text is a good idea.

anewton1998 commented:
I updated this PR to remove the explicit order, instead making it an agreement with the chairs and the AD. I also moved around the paragraphs on requirements since they were intermingled with this one.

jasdips commented on rpp-charter.md:
```diff
@@ -60,18 +60,27 @@ for the promotion of work to the IESG.
+IESG documents addressing the following topics in an order determined through agreement
+of the chairs with the responsible Area Director:
+* core architecture
+* specifications necessary to implement the core architecture
```
What's the difference between the "core architecture" and "specifications necessary to implement the core architecture" topics?

pawel-kow commented on rpp-charter.md:
```diff
@@ -60,18 +60,27 @@ for the promotion of work to the IESG.
 This working group will find consensus on a set of core requirements for RPP.
 In alignment with the [IESG's Statement on Support Documents dated 2023-08-24](https://datatracker.ietf.org/doc/statement-iesg-support-documents-in-ietf-working-groups-20230824/),
 these requirements will be publicly available on the working group's wiki.
+Analysis of the functionality included and commonly used in core EPP for domain names, 
+hosts and contacts (RFC5730,5731,5732,5733) may be created as input material for the requirements. 
```
I like the rearrangement but noticed the change from "will" to "may".
The discussion on the mailing list so far was rather going into direction that such analysis will be created as an input.

pawel-kow replied:
> If nobody is willing to do the analysis or there is no agreement on that analysis, the working group should not stall. This is an area of potential rabbit hole exploring. Thanks for noting this.

I think if nobody would be willing to, it would indicate other problems in the working group. The conclusion of the discussion so far was that without this input we may not be able to validate if requirements and architecture are actually correct and complete to fulfil other goals and deliverables.
The text does not mean that there must be only one or more, in which form and even not if there must be a consensus around the analysis. And this is meant like this. More perspectives are typically better on an early stage.

The requirements is where we will be looking after consensus. With analysis done we may expect better quality of requirements coming out of the process.

rpp-ietf commented:
I don’t understand why there would be any debate related to the value of analyzing the features and extensions that exist in the current provisioning protocol for the creation of a new one.  The analysis will be input into the requirements.  The result should be a set of recommendations for elements in EPP to use, not use, and to redo based on practical experience.  Volunteers need to be requested, and my hope is that the experienced EPP technologists will be happy to participate in this analysis.  I certainly would like to be involved in the effort.  This is a MUST and not a MAY in my opinion.

--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<http://verisigninc.com/>

From: Pawel Kowalik ***@***.***>
Reply-To: SIDN/ietf-wg-rpp-charter ***@***.***>
Date: Thursday, November 21, 2024 at 5:04 PM
To: SIDN/ietf-wg-rpp-charter ***@***.***>
Cc: rpp-ietf ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


@pawel-kow commented on this pull request.

________________________________

In rpp-charter.md<https://secure-web.cisco.com/1BwGpsdn2N9zGq1_u77raDRMSSXyQucvougs687FD-fqyIDhahLbMhZowQpZxNdVPzaQwCsX9M5EQysmZcl22kaKyBkbuimIkyLmIJE6uZ6Xoj3C8o5cPG9qpccPvKvjmxKYj47tnU-RBHlYxd4DllLps-VgdNONlS-tTxJgm9JR1FfJTefh3oV8b1K17PeEqUUJvmmykMfTE45AjTPcc-s_DSsZ_Dzeg_jPWAJ4lzpzziEMIGDsljYa60WaoNLT_zuwVTIhosE2fGnYnkWEc-HILfSZwjRuq9MD5z32lc6w/https%3A%2F%2Fgithub.com%2FSIDN%2Fietf-wg-rpp-charter%2Fpull%2F40%23discussion_r1852969645>:

> +Analysis of the functionality included and commonly used in core EPP for domain names,

+hosts and contacts (RFC5730,5731,5732,5733) may be created as input material for the requirements.

If nobody is willing to do the analysis or there is no agreement on that analysis, the working group should not stall. This is an area of potential rabbit hole exploring. Thanks for noting this.

I think if nobody would be willing to, it would indicate other problems in the working group. The conclusion of the discussion so far was that without this input we may not be able to validate if requirements and architecture are actually correct and complete to fulfil other goals and deliverables.
The text does not mean that there must be only one or more, in which form and even not if there must be a consensus around the analysis. And this is meant like this. More perspectives are typically better on an early stage.

The requirements is where we will be looking after consensus. With analysis done we may expect better quality of requirements coming out of the process.

—
Reply to this email directly, view it on GitHub<https://secure-web.cisco.com/1BwGpsdn2N9zGq1_u77raDRMSSXyQucvougs687FD-fqyIDhahLbMhZowQpZxNdVPzaQwCsX9M5EQysmZcl22kaKyBkbuimIkyLmIJE6uZ6Xoj3C8o5cPG9qpccPvKvjmxKYj47tnU-RBHlYxd4DllLps-VgdNONlS-tTxJgm9JR1FfJTefh3oV8b1K17PeEqUUJvmmykMfTE45AjTPcc-s_DSsZ_Dzeg_jPWAJ4lzpzziEMIGDsljYa60WaoNLT_zuwVTIhosE2fGnYnkWEc-HILfSZwjRuq9MD5z32lc6w/https%3A%2F%2Fgithub.com%2FSIDN%2Fietf-wg-rpp-charter%2Fpull%2F40%23discussion_r1852969645>, or unsubscribe<https://secure-web.cisco.com/12XBMiZH6hBkvcmNpDwrNIdawggx9_kznni8hp6CZhqc2-hqJVCWOIU628dhrG-PAFt2PqqcSPICE_59mUAO0QV_Bq0VUyjlPiUvmE_xGznmrY6vTl1YfPW1TnRT7z-_J9uY3DkXiewInl3T5quTIgYlADYxa-_vy1nWw_iRmP3jMuzkUY1tWmdYqX0I_NL_-mppMWX4qeX1dkg6tUqZLoFJPKF-ULiERyUlOUquLYYJsP-MoL0sqnrtR-8hEvhQqHaQh_Jphkf0EG-Ki5Z6sy7Z5qKebjuji1qNZo4AQ9pM/https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FBM5CYNOSJSTDJAS5I5AAU732BZKGHAVCNFSM6AAAAABSFQQ3O6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDINJSHA2DMNJZGM>.
You are receiving this because you commented.Message ID: ***@***.***>

anewton1998 commented:
I changed the wording to "is expected to be created". The concern is paralysis by analysis that could cause the working group to stall.

rpp-ietf commented:
Andy,

We can scope the analysis work.  The EPP RFCs are well structured and clear with many implementations and there is a registered set of EPP extensions in the EPP extension registry to evaluate.  My recommendation is if there are additional unregistered proprietary EPP extensions that should be considered, for the registries to ensure to get them registered in the EPP extension registry.

Thanks,

--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<http://verisigninc.com/>

From: Andrew Newton ***@***.***>
Reply-To: SIDN/ietf-wg-rpp-charter ***@***.***>
Date: Friday, November 22, 2024 at 9:15 AM
To: SIDN/ietf-wg-rpp-charter ***@***.***>
Cc: rpp-ietf ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


I changed the wording to "is expected to be created". The concern is paralysis by analysis that could cause the working group to stall.

—
Reply to this email directly, view it on GitHub<https://secure-web.cisco.com/16iA6ea-VFjM1SUHfDv_-SlB3vAEYt1PhqmBDxzfXhnPXAeRd5XUvuQxRGpG61m2ZW5pk0Qh5pRDmwF918FC7pEg9v9aUr_HNtSXj1QuPHwCbf39eb2jHbRteceSO1qVYOkwCd1t55fYChDN_4-CkHKS4ls-fLiL367DyVvEJMRtu8x0DhEfpNpM5bR9BgBEaQzWekUHcXGJHWpl38huFSof0x-VBp2vHfaNvt4wxy_eb45JAHe-JavSFTxPTD4S4Gmb_bEl6BTGXh4x9g8ghurs-M8cnelbJGy5SXwtA6LU/https%3A%2F%2Fgithub.com%2FSIDN%2Fietf-wg-rpp-charter%2Fpull%2F40%23issuecomment-2493864351>, or unsubscribe<https://secure-web.cisco.com/1cYXZZGtoxH0H_PtGmTHQRsMDjwVnry0Z1BQpUEPw9K26p7BMUKAsR4xj9hmiTQmnMBIitOnQ5wHWZjRghwlREJKGoupD3ErdPKaj3RvBrHhkKZnq2_HApcIzJNNb91mojkrEhW3u_Kb9lVh4sesA8RAf4-tMItHgaStudHrVFDbRWMZu_4Se956qcJs2iB28GKeCeswNVinIFEzjnF0LYhlXUF3gZFKjTrOVHmr0nDBY6rz2gGMge85TgGh8-Yd0JgzWlH7P6V--QJklXflEssMPd67Yqt8tudx-cwfCtNw/https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FBM5CYNP6DWRPVFJ2366CC6T2B44BLAVCNFSM6AAAAABSFQQ3O6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIOJTHA3DIMZVGE>.
You are receiving this because you commented.Message ID: ***@***.***>

anewton1998 commented:
There is nothing stopping anybody from starting that work now. :)

rpp-ietf commented:
Nope, who are the volunteers?  It sounds like we have two thus far with me and Pawel (“kind of”, which I take as a “yes”).

--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<http://verisigninc.com/>

From: Andrew Newton ***@***.***>
Reply-To: SIDN/ietf-wg-rpp-charter ***@***.***>
Date: Friday, November 22, 2024 at 10:07 AM
To: SIDN/ietf-wg-rpp-charter ***@***.***>
Cc: rpp-ietf ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


There is nothing stopping anybody from starting that work now. :)

—
Reply to this email directly, view it on GitHub<https://secure-web.cisco.com/1CADewsi1b41V40qnW__MVHSjgN8OI0a-QYLhlwpynAqRJjnmGODn4d9AaWFpHuSKPU_RnHiOQt_jVc8UCL_3bvOXMPtpi3m4H3JhSQA7_lNOD478ZBRZMASWY9MbaYb2G1rULX6wygG1QFkvmiuKWNJ9xuhdhcHAQnrbih4gXtCsMdnTP-9R2L2nezirQXzpe-CDo6XvqLg4IKZMCUMdnt1KihPSsNxiX-9BjNybqbjosYSVWk7JuSWJ8qsXgH7ffOoKvWtw2CZMAFxEwTpfjLtA3f0SkyQqCDKeQXQXCTg/https%3A%2F%2Fgithub.com%2FSIDN%2Fietf-wg-rpp-charter%2Fpull%2F40%23issuecomment-2493969790>, or unsubscribe<https://secure-web.cisco.com/1E91U-hhFhbfu7XqqauynyTJNWiJZnBjDyWk2hggFvD_RDXWbxLagUQuc45HVZmQOeJwoghoDMx3Aol0Fgpnu32oXGM1t5KFA7kt0nUXnvuw4a4qrn4jzCRIYMfcH8g__W6GJFhyG6dKA4H7zq2XsSWI3f1it9VgaCWcGkQOlBSirdRFoYvvIMbxZngk0cTkuiU9fNYQ95h3AxPImjVdYxxF8egUvShpD0IxbyfjuCEVpb13MwtVDWXmp8c7rL5ZSKSUSQSNM3v_LI79r0gJ1DlKx5YvZ-wcVSVNiKgE01GE/https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FBM5CYNOXTEWWATZ63DO5HV32B5CBRAVCNFSM6AAAAABSFQQ3O6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIOJTHE3DSNZZGA>.
You are receiving this because you commented.Message ID: ***@***.***>

rpp-ietf commented:
I can help out with this also.

Thanks,
Jody Kolker
319-329-9805  (mobile)

Please contact my direct supervisor Scott Courtney ***@***.******@***.***>) with any feedback.

This email message and any attachments hereto is intended for use only by the addressee(s) named herein and may contain confidential information. If you have received this email in error, please immediately notify the sender and permanently delete the original and any copy of this message and its attachments.

From: Gould, James ***@***.***>
Sent: Friday, November 22, 2024 10:52 AM
To: ***@***.***; ***@***.***
Cc: ***@***.***; ***@***.***
Subject: [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)

Nope, who are the volunteers? It sounds like we have two thus far with me and Pawel (“kind of”, which I take as a “yes”). -- JG James Gould Fellow Engineer jgould@ Verisign. com 703-948-3271 12061 Bluemont Way Reston, VA 20190 Verisign. com From: 
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
ZjQcmQRYFpfptBannerEnd
Nope, who are the volunteers?  It sounds like we have two thus far with me and Pawel (“kind of”, which I take as a “yes”).

--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<https://urldefense.com/v3/__http:/verisigninc.com/__;!!Hj18uoVe_Lnx!qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojASHy609$>

From: Andrew Newton ***@***.******@***.***>>
Reply-To: SIDN/ietf-wg-rpp-charter ***@***.******@***.***>>
Date: Friday, November 22, 2024 at 10:07 AM
To: SIDN/ietf-wg-rpp-charter ***@***.******@***.***>>
Cc: rpp-ietf ***@***.******@***.***>>, Comment ***@***.******@***.***>>
Subject: [EXTERNAL] [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

There is nothing stopping anybody from starting that work now. :)

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/secure-web.cisco.com/1CADewsi1b41V40qnW__MVHSjgN8OI0a-QYLhlwpynAqRJjnmGODn4d9AaWFpHuSKPU_RnHiOQt_jVc8UCL_3bvOXMPtpi3m4H3JhSQA7_lNOD478ZBRZMASWY9MbaYb2G1rULX6wygG1QFkvmiuKWNJ9xuhdhcHAQnrbih4gXtCsMdnTP-9R2L2nezirQXzpe-CDo6XvqLg4IKZMCUMdnt1KihPSsNxiX-9BjNybqbjosYSVWk7JuSWJ8qsXgH7ffOoKvWtw2CZMAFxEwTpfjLtA3f0SkyQqCDKeQXQXCTg/https*3A*2F*2Fgithub.com*2FSIDN*2Fietf-wg-rpp-charter*2Fpull*2F40*23issuecomment-2493969790__;JSUlJSUlJSU!!Hj18uoVe_Lnx!qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojCJKMPXG$>, or unsubscribe<https://urldefense.com/v3/__https:/secure-web.cisco.com/1E91U-hhFhbfu7XqqauynyTJNWiJZnBjDyWk2hggFvD_RDXWbxLagUQuc45HVZmQOeJwoghoDMx3Aol0Fgpnu32oXGM1t5KFA7kt0nUXnvuw4a4qrn4jzCRIYMfcH8g__W6GJFhyG6dKA4H7zq2XsSWI3f1it9VgaCWcGkQOlBSirdRFoYvvIMbxZngk0cTkuiU9fNYQ95h3AxPImjVdYxxF8egUvShpD0IxbyfjuCEVpb13MwtVDWXmp8c7rL5ZSKSUSQSNM3v_LI79r0gJ1DlKx5YvZ-wcVSVNiKgE01GE/https*3A*2F*2Fgithub.com*2Fnotifications*2Funsubscribe-auth*2FBM5CYNOXTEWWATZ63DO5HV32B5CBRAVCNFSM6AAAAABSFQQ3O6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIOJTHE3DSNZZGA__;JSUlJSUl!!Hj18uoVe_Lnx!qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojI0kttWD$>.
You are receiving this because you commented.Message ID: ***@***.******@***.***>>

rpp-ietf commented:
Jody,

Thanks for volunteering!  Any more volunteers?  We currently have me, Pawel, and Jody on the tiger team to perform the EPP extensibility and extensions analysis.  Once the tiger team is formed, I can setup a time for us to meet to discuss the analysis approach and the split out of the work.  I believe the wiki is the best place to capture this.  This work can be reflected as a deliverable in the charter.  How about updating the start of the Deliverables section of the Charter to read?

This working group will publish on its wiki its operational practices,
including but not limited to, the adoption of work items, usage of
tools beyond the IETF’s datatracker, analysis of EPP extensibility and the existing EPP extensions,
and implementation requirements for the promotion of work to the IESG.


--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<http://verisigninc.com/>

From: Jody Kolker ***@***.***>
Date: Wednesday, December 4, 2024 at 8:18 AM
To: James Gould ***@***.***>, ***@***.***" ***@***.***>, ***@***.***" ***@***.***>
Cc: ***@***.***" ***@***.***>, ***@***.***" ***@***.***>
Subject: [EXTERNAL] RE: [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

I can help out with this also.

Thanks,
Jody Kolker
319-329-9805  (mobile)

Please contact my direct supervisor Scott Courtney ***@***.******@***.***>) with any feedback.

This email message and any attachments hereto is intended for use only by the addressee(s) named herein and may contain confidential information. If you have received this email in error, please immediately notify the sender and permanently delete the original and any copy of this message and its attachments.

From: Gould, James ***@***.***>
Sent: Friday, November 22, 2024 10:52 AM
To: ***@***.***; ***@***.***
Cc: ***@***.***; ***@***.***
Subject: [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)

Nope, who are the volunteers? It sounds like we have two thus far with me and Pawel (“kind of”, which I take as a “yes”). -- JG James Gould Fellow Engineer jgould@ Verisign. com 703-948-3271 12061 Bluemont Way Reston, VA 20190 Verisign. com From: 
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender

This message came from outside your organization.



ZjQcmQRYFpfptBannerEnd
Nope, who are the volunteers?  It sounds like we have two thus far with me and Pawel (“kind of”, which I take as a “yes”).

--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<https://secure-web.cisco.com/1NkINdsoI0K9-fQz4EaOqiVWwn-COLYC4D7V4ng3ptlmpGIbI-qnxzLaXUvN8u6uscEDeiHHze5nNugzFB6jcR1meImNMYLtQMlefvz12sbMi185ASWy3Ozvk6mkT4vQqMTzv2voYPKWrCjlWcVqIAXMyZeEBAtPkWwedgVL2MLz26PPyNJxF1dSpD76GoI_QQiAf1FOQnM56up3bAWxac47RcqKjW2NW-l9vexXeDutLxhABYFdAYl1P6fDsrvuFeaP4ObraVHhOWOU3zXuuQjv5cp7LcC3MUkgmjrIJFpM/https%3A%2F%2Furldefense.com%2Fv3%2F__http%3A%2Fverisigninc.com%2F__%3B%21%21Hj18uoVe_Lnx%21qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojASHy609%24>

From: Andrew Newton ***@***.******@***.***>>
Reply-To: SIDN/ietf-wg-rpp-charter ***@***.******@***.***>>
Date: Friday, November 22, 2024 at 10:07 AM
To: SIDN/ietf-wg-rpp-charter ***@***.******@***.***>>
Cc: rpp-ietf ***@***.******@***.***>>, Comment ***@***.******@***.***>>
Subject: [EXTERNAL] [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


There is nothing stopping anybody from starting that work now. :)

—
Reply to this email directly, view it on GitHub<https://secure-web.cisco.com/1dw8k-UH_XvJHqWtgnvDC554z9NAJut-S18I2incRn0SjWq_J4rzyTYmcxr30yFHehigHMXSs5aMKVZonskcFQ7H6YJ40L4JbbpdM9HkTTIq9p-IRsgohWEVAsZeWtQ-ONQDTg7woXfS3ALlXJAMt29MSRsK6PGr-FfhYuo1-9vtTfno2OmMBx4jCpNsBZEw0jLj3NzOKSEJA5AvGheu-n4WXgNlm9jCN-TSANMeDx1c63qlktrodm_jSVanhgM7wNvZi39qOmjp7cvf3TNmjJz-unQMf_9uk-bciJ1iIeaU/https%3A%2F%2Furldefense.com%2Fv3%2F__https%3A%2Fsecure-web.cisco.com%2F1CADewsi1b41V40qnW__MVHSjgN8OI0a-QYLhlwpynAqRJjnmGODn4d9AaWFpHuSKPU_RnHiOQt_jVc8UCL_3bvOXMPtpi3m4H3JhSQA7_lNOD478ZBRZMASWY9MbaYb2G1rULX6wygG1QFkvmiuKWNJ9xuhdhcHAQnrbih4gXtCsMdnTP-9R2L2nezirQXzpe-CDo6XvqLg4IKZMCUMdnt1KihPSsNxiX-9BjNybqbjosYSVWk7JuSWJ8qsXgH7ffOoKvWtw2CZMAFxEwTpfjLtA3f0SkyQqCDKeQXQXCTg%2Fhttps%2A3A%2A2F%2A2Fgithub.com%2A2FSIDN%2A2Fietf-wg-rpp-charter%2A2Fpull%2A2F40%2A23issuecomment-2493969790__%3BJSUlJSUlJSU%21%21Hj18uoVe_Lnx%21qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojCJKMPXG%24>, or unsubscribe<https://secure-web.cisco.com/1KDIKWcOy1NDiAlGp7huq9NhgDYb7H7elqXpX6dO4E3mf4KQzDbXbzgzuY9qXwsfI0x0OY_cknyYQifVzCF8pd9GsUu9kntS7yHJJarr24sQrPZdb03bHHJgJS96vyPMo36LjhqHMvcDwGgGpSnbhU9qS6tBgh19vo9vZKnan0oL40EIBQ14i6g7ePuAqBz_BHaR16DZL7lNc01oOKvda7yVGhFxyUrqSJwiKFD06tNrM-GUtlwtRh48cDL0CowgUjnTynbrll_dMsu4p_XAwL-k9QD2r2wZT2lLMlMhBkOY/https%3A%2F%2Furldefense.com%2Fv3%2F__https%3A%2Fsecure-web.cisco.com%2F1E91U-hhFhbfu7XqqauynyTJNWiJZnBjDyWk2hggFvD_RDXWbxLagUQuc45HVZmQOeJwoghoDMx3Aol0Fgpnu32oXGM1t5KFA7kt0nUXnvuw4a4qrn4jzCRIYMfcH8g__W6GJFhyG6dKA4H7zq2XsSWI3f1it9VgaCWcGkQOlBSirdRFoYvvIMbxZngk0cTkuiU9fNYQ95h3AxPImjVdYxxF8egUvShpD0IxbyfjuCEVpb13MwtVDWXmp8c7rL5ZSKSUSQSNM3v_LI79r0gJ1DlKx5YvZ-wcVSVNiKgE01GE%2Fhttps%2A3A%2A2F%2A2Fgithub.com%2A2Fnotifications%2A2Funsubscribe-auth%2A2FBM5CYNOXTEWWATZ63DO5HV32B5CBRAVCNFSM6AAAAABSFQQ3O6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIOJTHE3DSNZZGA__%3BJSUlJSUl%21%21Hj18uoVe_Lnx%21qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojI0kttWD%24>.
You are receiving this because you commented.Message ID: ***@***.******@***.***>>

mwullink commented:
Count me in also.

-
Maarten

Op 4 dec 2024, om 14:36 heeft rpp-ietf ***@***.***> het volgende geschreven:


Jody,

Thanks for volunteering! Any more volunteers? We currently have me, Pawel, and Jody on the tiger team to perform the EPP extensibility and extensions analysis. Once the tiger team is formed, I can setup a time for us to meet to discuss the analysis approach and the split out of the work. I believe the wiki is the best place to capture this. This work can be reflected as a deliverable in the charter. How about updating the start of the Deliverables section of the Charter to read?

This working group will publish on its wiki its operational practices,
including but not limited to, the adoption of work items, usage of
tools beyond the IETF’s datatracker, analysis of EPP extensibility and the existing EPP extensions,
and implementation requirements for the promotion of work to the IESG.


--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<http://verisigninc.com/>

From: Jody Kolker ***@***.***>
Date: Wednesday, December 4, 2024 at 8:18 AM
To: James Gould ***@***.***>, ***@***.***" ***@***.***>, ***@***.***" ***@***.***>
Cc: ***@***.***" ***@***.***>, ***@***.***" ***@***.***>
Subject: [EXTERNAL] RE: [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

I can help out with this also.

Thanks,
Jody Kolker
319-329-9805 (mobile)

Please contact my direct supervisor Scott Courtney ***@***.******@***.***>) with any feedback.

This email message and any attachments hereto is intended for use only by the addressee(s) named herein and may contain confidential information. If you have received this email in error, please immediately notify the sender and permanently delete the original and any copy of this message and its attachments.

From: Gould, James ***@***.***>
Sent: Friday, November 22, 2024 10:52 AM
To: ***@***.***; ***@***.***
Cc: ***@***.***; ***@***.***
Subject: [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)

Nope, who are the volunteers? It sounds like we have two thus far with me and Pawel (“kind of”, which I take as a “yes”). -- JG James Gould Fellow Engineer jgould@ Verisign. com 703-948-3271 12061 Bluemont Way Reston, VA 20190 Verisign. com From: 
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender

This message came from outside your organization.



ZjQcmQRYFpfptBannerEnd
Nope, who are the volunteers? It sounds like we have two thus far with me and Pawel (“kind of”, which I take as a “yes”).

--

JG

***@***.***

James Gould
Fellow Engineer
***@***.******@***.***>

703-948-3271
12061 Bluemont Way
Reston, VA 20190

Verisign.com<https://secure-web.cisco.com/1NkINdsoI0K9-fQz4EaOqiVWwn-COLYC4D7V4ng3ptlmpGIbI-qnxzLaXUvN8u6uscEDeiHHze5nNugzFB6jcR1meImNMYLtQMlefvz12sbMi185ASWy3Ozvk6mkT4vQqMTzv2voYPKWrCjlWcVqIAXMyZeEBAtPkWwedgVL2MLz26PPyNJxF1dSpD76GoI_QQiAf1FOQnM56up3bAWxac47RcqKjW2NW-l9vexXeDutLxhABYFdAYl1P6fDsrvuFeaP4ObraVHhOWOU3zXuuQjv5cp7LcC3MUkgmjrIJFpM/https%3A%2F%2Furldefense.com%2Fv3%2F__http%3A%2Fverisigninc.com%2F__%3B%21%21Hj18uoVe_Lnx%21qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojASHy609%24>

From: Andrew Newton ***@***.******@***.***>>
Reply-To: SIDN/ietf-wg-rpp-charter ***@***.******@***.***>>
Date: Friday, November 22, 2024 at 10:07 AM
To: SIDN/ietf-wg-rpp-charter ***@***.******@***.***>>
Cc: rpp-ietf ***@***.******@***.***>>, Comment ***@***.******@***.***>>
Subject: [EXTERNAL] [rpp] Re: [SIDN/ietf-wg-rpp-charter] addressing issue #26 (mapping RPP and EPP as deliverable) (PR #40)


Caution: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


There is nothing stopping anybody from starting that work now. :)

—
Reply to this email directly, view it on GitHub<https://secure-web.cisco.com/1dw8k-UH_XvJHqWtgnvDC554z9NAJut-S18I2incRn0SjWq_J4rzyTYmcxr30yFHehigHMXSs5aMKVZonskcFQ7H6YJ40L4JbbpdM9HkTTIq9p-IRsgohWEVAsZeWtQ-ONQDTg7woXfS3ALlXJAMt29MSRsK6PGr-FfhYuo1-9vtTfno2OmMBx4jCpNsBZEw0jLj3NzOKSEJA5AvGheu-n4WXgNlm9jCN-TSANMeDx1c63qlktrodm_jSVanhgM7wNvZi39qOmjp7cvf3TNmjJz-unQMf_9uk-bciJ1iIeaU/https%3A%2F%2Furldefense.com%2Fv3%2F__https%3A%2Fsecure-web.cisco.com%2F1CADewsi1b41V40qnW__MVHSjgN8OI0a-QYLhlwpynAqRJjnmGODn4d9AaWFpHuSKPU_RnHiOQt_jVc8UCL_3bvOXMPtpi3m4H3JhSQA7_lNOD478ZBRZMASWY9MbaYb2G1rULX6wygG1QFkvmiuKWNJ9xuhdhcHAQnrbih4gXtCsMdnTP-9R2L2nezirQXzpe-CDo6XvqLg4IKZMCUMdnt1KihPSsNxiX-9BjNybqbjosYSVWk7JuSWJ8qsXgH7ffOoKvWtw2CZMAFxEwTpfjLtA3f0SkyQqCDKeQXQXCTg%2Fhttps%2A3A%2A2F%2A2Fgithub.com%2A2FSIDN%2A2Fietf-wg-rpp-charter%2A2Fpull%2A2F40%2A23issuecomment-2493969790__%3BJSUlJSUlJSU%21%21Hj18uoVe_Lnx%21qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojCJKMPXG%24>, or unsubscribe<https://secure-web.cisco.com/1KDIKWcOy1NDiAlGp7huq9NhgDYb7H7elqXpX6dO4E3mf4KQzDbXbzgzuY9qXwsfI0x0OY_cknyYQifVzCF8pd9GsUu9kntS7yHJJarr24sQrPZdb03bHHJgJS96vyPMo36LjhqHMvcDwGgGpSnbhU9qS6tBgh19vo9vZKnan0oL40EIBQ14i6g7ePuAqBz_BHaR16DZL7lNc01oOKvda7yVGhFxyUrqSJwiKFD06tNrM-GUtlwtRh48cDL0CowgUjnTynbrll_dMsu4p_XAwL-k9QD2r2wZT2lLMlMhBkOY/https%3A%2F%2Furldefense.com%2Fv3%2F__https%3A%2Fsecure-web.cisco.com%2F1E91U-hhFhbfu7XqqauynyTJNWiJZnBjDyWk2hggFvD_RDXWbxLagUQuc45HVZmQOeJwoghoDMx3Aol0Fgpnu32oXGM1t5KFA7kt0nUXnvuw4a4qrn4jzCRIYMfcH8g__W6GJFhyG6dKA4H7zq2XsSWI3f1it9VgaCWcGkQOlBSirdRFoYvvIMbxZngk0cTkuiU9fNYQ95h3AxPImjVdYxxF8egUvShpD0IxbyfjuCEVpb13MwtVDWXmp8c7rL5ZSKSUSQSNM3v_LI79r0gJ1DlKx5YvZ-wcVSVNiKgE01GE%2Fhttps%2A3A%2A2F%2A2Fgithub.com%2A2Fnotifications%2A2Funsubscribe-auth%2A2FBM5CYNOXTEWWATZ63DO5HV32B5CBRAVCNFSM6AAAAABSFQQ3O6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIOJTHE3DSNZZGA__%3BJSUlJSUl%21%21Hj18uoVe_Lnx%21qezaWBMKflaHLRuh8tcwI3itKgXMzPYbdF_eDCgpXVXisGv-jTVUoD6cOxNG9fqdom83MeQuhZcFhiYvXXSnFJvojI0kttWD%24>.
You are receiving this because you commented.Message ID: ***@***.******@***.***>>

—
Reply to this email directly, view it on GitHub<https://github.com/SIDN/ietf-wg-rpp-charter/pull/40#issuecomment-2517423051>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AALSFIG7MSRWFRW3GNGATG32D4APJAVCNFSM6AAAAABSFQQ3O6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKMJXGQZDGMBVGE>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
